src/Writer/Deleted.php

```diff
@@ -32,12 +32,13 @@ class Deleted
     /**
      * Set the feed character encoding
      *
-     * @param  string $encoding
-     * @return null|string
-     * @return Deleted
+     * @param null|string $encoding
+     *
+     * @return self
+     *
      * @throws Exception\InvalidArgumentException
      */
-    public function setEncoding($encoding)
+    public function setEncoding(?string $encoding): self
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.